### ISSUE TITLE
crash on updateConstraints()

### ISSUE THREAD

**Issue by johnpaulmanoza** [AssistoLab/DropDown#28]
Hi, I tried using this control to one of our apps and it crashes the app. I received this crash report. I have no clue what causing the crash. Hoping for any helps. Thank you.

<img width="1280" alt="screen shot 2016-07-18 at 10 37 39 am" src="https://cloud.githubusercontent.com/assets/2209713/16905134/fdc9b14c-4cd3-11e6-9ef6-049127b6eeca.png">

**Comment by kevin-hirsch** [comment]
Hello @johnpaulmanoza!
Could you show me the code where you are calling the drop down? :)

**Comment by kevin-hirsch** [comment]
It is probably the same issue as #20 
You can check my answer that fixed this issue [here](https://github.com/AssistoLab/DropDown/issues/20#issuecomment-225595458)
Does that help? 🙃

**Comment by Dfmarulanda** [comment]
Is not the same issue as #20 
I'm facing the same issue, this is my code:

```swift
class MapViewController: UIViewController{
let dropDown = Dropdown()

    override func viewDidLoad() {
        dropDown.dataSource = ["Profile".localized, "Logout".localized]
        dropDown.anchorView = self.openItemsButton
   }

    @IBAction func optionsButtonPressed(sender: AnyObject) {
        dropDown.selectionAction = {  (index: Int, item: String) in
            switch index {
            case 0:
              print("case_0")
              break
            case 1:
              print("case_1")
              break
            default:
              print("case_default")
              break
         }
       }
    }
}
```


I managed to fix it changing line 485 in DropDown.swift file and removing the unowned self declaration

```swift
	DispatchQueue.main.async { _ in
         	self.tableView.flashScrollIndicators()
	}

```

**Comment by kevin-hirsch** [comment]
Hello!
I've pushed a new commit that should fix this problem (80a125b on master)
